tree/BinarySearchTree.js

```diff
@@ -0,0 +1,57 @@
+
+const Node = require('./Node');
+
+class BST {
+  constructor(root = null){
+    this.root = root;
+  }
+
+  add(value){
+    const node = this.root;
+    if(node === null) {
+      this.root = new Node(value);
+      return;
+    } else {
+      const searchTree = function(node){
```

Comment:
It's not necessary to define, and then call, a function in your else. You are already expecting to receive a node as an argument to your add method, and can use that node in your logic here.